build.sbt

```diff
@@ -33,7 +34,8 @@ lazy val scalaYamlCore = crossProject(JSPlatform, JVMPlatform)
     name              := projectName,
     scalaVersion      := scala3Version,
     semanticdbVersion := scalafixSemanticdb.revision,
-    semanticdbEnabled := true
+    semanticdbEnabled := true,
+    libraryDependencies ++= Seq(Deps.pprint % Test)
   )
   .settings(docsSettings)
```

Comment:
I think, you can add here `.settings(testSettings)`

Reply:
`testSettings`, except dependencies, contains also `publish / skip := true` so it cannot be added.